gui/qt/main_window.py

```diff
@@ -1385,7 +1400,11 @@ def do_send(self, preview = False):
             return
         outputs, fee, tx_desc, coins = r
         try:
-            tx = self.wallet.make_unsigned_transaction(coins, outputs, self.config, fee)
+            opreturn_message = self.message_opreturn_e.text().encode('utf-8').hex()
+            if len(opreturn_message)>440:
```

Comment:
This check is not present in the chunk above.

How about a helper function that takes a list of outputs, and adds an op return message to them?  All the logic is in one place and make_unsigned_transaction doesn't need to change

Reply:
feel like refactoring this?